### DISCUSSION TITLE
How can I get slack notifications Only when site is down?

### DISCUSSION THREAD

**Discussion by SG2019** [upptime/upptime#598] [Q&A] [answered]
I have integrated with Slack using webhook URL. We are getting all the notifications in Slack channel even when there is a change by upptime-bot.

Is it possible to get slack notification only when endpoint is down i.e. when issue gets created to reduce the noise in channel?

Thankyou

**Comment by AnandChowdhary** [comment]
You should only get them when a site is down and when it comes back up. Can you share more about how frequently and what notifies you currently?

**Comment by SG2019** [comment]
Thankyou @AnandChowdhary.
We are getting commits notifications to master and github-pages. The time is in AEST (UTC +10)
We normally get these notifications 4-5 times a day.
![image](https://user-images.githubusercontent.com/49736371/174196811-200a394a-c702-41cf-8c7d-c107f31dfaf2.png)

**Reply by AnandChowdhary** [reply]
These notifications are unrelated to Upptime. You have enabled the GItHub Slack app which notifies you of every commit and deployment.

**Reply by SG2019** [reply]
Thankyou @AnandChowdhary.

I have added slack environment variables in repository secrets and unsubscribed from github app from slack channel. 
Hopefully these are the only steps to be done to get Upptime notifications

**Comment by SG2019** [comment, marked as answer]
We have tested and notifications from Upptime are working now. Thankyou @AnandChowdhary 

Just one thing that the issue that was created automatically when the site was down , got deleted after the site was back. Is this expected? Do we need to change any variable to stop auto deleting issues and just close the issue?

**Reply by AnandChowdhary** [reply]
Please see https://github.com/upptime/upptime/discussions/492
